static/app/views/acceptProjectTransfer/index.tsx

```diff
@@ -42,8 +42,10 @@ class AcceptProjectTransfer extends DeprecatedAsyncView<Props, State> {
       },
       success: () => {
         const orgSlug = formData.organization;
-
-        this.props.router.push(normalizeUrl(`/organizations/${orgSlug}/projects/`));
+        const projectSlug = this.state.transferDetails.project.slug;
+        this.props.router.push(
+          normalizeUrl(`/settings/${orgSlug}/projects/${projectSlug}/teams/`)
```

Comment:
This was tough to test locally since we can't create new organizations - I went directly to the `accept-transfer` endpoint to accept the transfer, but it's not actually doing anything since it's "transferring" a project to the org it already belongs in. 

Testing this locally I was surprised to see that after accepting the project transfer, I'm not redirected to the project's new organization, which is what I'd expect to happen based on the existing code.